ts/components/dialog/conversationSettings/conversationSettingsItems.tsx

```diff
@@ -175,6 +175,8 @@ export const PinUnpinButton = ({ conversationId }: WithConvoId) => {
   const showPinUnpin = useShowPinUnpin(conversationId);
   const isPinned = useIsPinned(conversationId);
 
+  const togglePinConversation = useTogglePinConversationHandler(conversationId);
+
   if (!showPinUnpin) {
     return null;
```

Comment:
see my comment below, but this
`const showPinUnpin = useShowPinUnpin(conversationId);` should be removed.

`if (!showPinUnpin) {` should be replaced with `if(!togglePinConversation)` to handle the null case (see my comment in that file)